receipt_dynamo/receipt_dynamo/utils/geospatial.py

```
@@ -0,0 +1,247 @@
+
+        # Get bounding box for 1km around NYC
+        min_lat, max_lat, min_lng, max_lng = bounding_box_from_radius(
+            40.7128, -74.0060, radius_km=1.0
+        )
+        # Use to filter: latitude >= min_lat AND latitude <= max_lat, etc.
+        ```
+    """
+    # Approximate degrees per km at Earth's surface
+    # More precise near equator, less precise near poles
+    DEGREES_PER_KM_LAT = 1 / 111.32  # Latitude (always ~111km per degree)
+    DEGREES_PER_KM_LNG = 1 / 111.32 * cos(radians(latitude))  # Longitude (varies by latitude)
```

Ruff (0.14.8)

Undefined name `cos`

(F821)